cli/flags/fixed8_test.go

```diff
@@ -50,14 +50,15 @@ func TestFixed8Flag_GetName(t *testing.T) {
 		Name: "myFlag",
 	}
 
-	require.Equal(t, "myFlag", flag.GetName())
+	require.Equal(t, []string{"myFlag"}, flag.Names())
 }
 
 func TestFixed8(t *testing.T) {
 	f := flag.NewFlagSet("", flag.ContinueOnError)
 	f.SetOutput(io.Discard) // don't pollute test output
-	gas := Fixed8Flag{Name: "gas, g"}
-	gas.Apply(f)
+	gas := Fixed8Flag{Name: "gas", Aliases: []string{"g"}, Usage: "Gas amount", Value: Fixed8{Value: 0}, Required: true, Hidden: false, Action: nil}
+	err := gas.Apply(f)
+	require.NoError(t, err)
```

Comment:
Please, extend this test with flag with missing value and required set to to true/false.